Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportApiHandler.cs

```diff
@@ -363,6 +364,81 @@ public async Task UploadImageAsync(Guid executionId, List<string> filePaths)
             }
         }
 
+        public async Task SendArtifactsToCentralDBAsync(Guid executionId, List<string> filePaths)
+        {
+            if (restClient != null)
+            {
+                if (filePaths != null && filePaths.Count > 0)
+                {
+                    try
+                    {
+                        Reporter.ToStatus(eStatusMsgKey.PublishingToCentralDB, null, "Uploading execution artifacts to central DB");
+
+                        long fileSize = 0;
+
+                        List<string> temp = new List<string>();
+
+                        foreach (string screenshot in filePaths)
```

Comment:
it's artifacts but code params are still "screenshot" - need to correct it

Reply:
Done.